tests/trafficmanagement/scripts/traffic_shifting.txt

```diff
@@ -18,6 +18,9 @@ set -e
 set -u
 set -o pipefail
 
+#TODO in what directory will the _snip.txt file be stored?
+source ${TEST_OUTPUT_DIR}/traffic-shifting_snip.txt
```

Comment:
We discussed `_snip.txt` being in the same directory as the test, which I think is fine. However, other tests may rely on these as well. For example, many tests rely on Bookinfo being present.

Reply:
I think Rigs suggested that the _snip file should be in the same directory as the md file. Since it's really just a tester's view of the doc, that seems like the best place to me too. We just need a variable that points to the doc root directory so that the test can then import something like this:
```
source ${DOC_ROOT_DIR}/docs/tasks/traffic-management/traffic-shifting/traffic-shifting_snip.txt
```